### ISSUE TITLE
Application review criteria for 3rd instructor training

### ISSUE THREAD

**Issue by mjcollin** [UF-Carpentry/Coordination#4]
The 3rd instructor training is going to use applications for selecting who attends. The Carpentry training request form will be used: https://amy.software-carpentry.org/forms/request_training/

With this information, what characteristics of applicants should we prioritize?

**Comment by fmichonneau** [comment]
I wanted to let you know that I added text and link to the application form on the workshop website: https://fmichonneau.github.io/2018-03-05-UF-ttt/

**Comment by fmichonneau** [comment]
One of the criteria to consider is affiliation. Is it open to anyone, will UF-affiliated people have priority, or is it strictly reserved for UF-affiliated people?

**Comment by mjcollin** [comment]
RE: affiliation, I just had that question and I referred the asker to our club charter:

"UF has a wide network of affiliated organizations and has strong relationships with many other universities and colleges in Florida. The club will leverage these connections to help other entities become a part of the Carpentry community.​"

I think it would be good to have some participants be people we hope will expand the community in FL.

**Comment by fmichonneau** [comment]
should we ask Deb to advertise at FSU? I think there were people interested in having a pool of instructors at USF as well. We can also hold off until we have a better idea of the local demand.

**Comment by mjcollin** [comment]
Deb was the "asker" from above :) She's forwarding the email to her usual suspects. I think it's best to see the largest possible pool of applicants so we can justify asking for more funds for additional trainings if that's needed.

**Comment by picardis** [comment]
Would you like me to find a way to advertise to SoFlo universities as well? (Nova, FIU, FAU)

**Comment by mjcollin** [comment]
I arrived at the process:
I did my scoring on another spreadsheet. I have people 1, 2, or 3 points (no, maybe/a bit, definitely) based on whether they were from a different location or discipline, whether they have been to a carpentries before, and my personal guess at how likely they would be to teach for us. I also gave people a 3 point bonus if they were on the waitlist. I found that a cutoff of 7 or greater resulted in 29 candidates and 8 or greater resulted in 14.

​Others seemed to use a similar weighting. However, this didn't seem to result in convergence though and we should try something else next time. From Kristina:

The bad news is that we all didn't really agree on applicants. Only 4 applicants got yes from all of us, then 13 applicants got yes from 5 of us and 6 applicants got yes from 4 of us. That would be a total of 23 applicants who got a yes from at least half of the board members. Because we've already sunk more than enough time into this process, I'm inclined to give those 23 people the chance to register.
